raiden/utils/typing.py

```diff
@@ -117,6 +117,8 @@
 T_Signature = bytes
 Signature = NewType('Signature', T_Signature)
 
+SignedBlindedBalanceProof = TypeVar('SignedBlindedBalanceProof', bound='SignedBlindedBalanceProof')
```

Comment:
Why use a `TypeVar` here? We don't use them at all across the codebase yet. What is the benefit and the thinking behind it?

Looking at [the docs](https://docs.python.org/3/library/typing.html#typing.TypeVar) they are supposed to be used for generics. Which does not seem to apply in this case.

Reply:
The type of `cls` is not available inside the class definition. Apparently there is some consensus to use the `TypeVar` way to solve this: https://github.com/python/typing/issues/58#issuecomment-326240794

Reply:
Hah. That's a funny hack. But I see why you did this now.